src/headers.ts

```diff
@@ -467,140 +474,203 @@ export const inchi_StereoParity0D = new Enum({
  *************************************************/
 
 /*
-  Structure -> InChI
+    Structure -> InChI
 
-  GetINCHI()
-  GetStdINCHI()
-  GetINCHIEx()
+    GetINCHI()
+    GetStdINCHI()
+    GetINCHIEx()
```

Comment:
- Why are these changes present?
- Revert them
- Running `yarn pretty` may auto-fix it.
- If that doesn't happen, fix it manually.